### PR TITLE
Back to 36

### PR DESCRIPTION
**Pull request by bollwyvl**
Like #35, but doesn't try to go after python 3.7.

The osx chromedriver for stable is up, so this _might_ work....

### CODE REVIEW

**Comment by datakurre** [comment]
@bollwyvl I have no idea why "robotlab-tutorial" command fails on Windows. There is no visible error in logs. Would be able to figure out that one?

I try to fix those tests...

File: robotlab/src/robotlab/examples.py [hunk 1]
```diff
@@ -14,7 +14,7 @@ def copy_robotkernel_examples(dest=None):
 
 if __name__ == "__main__":
     if len(sys.argv) > 1:
-        dest = Path(sys.argv(1))
+        dest = Path(sys.argv[1])
```
Comment (bollwyvl) [inline robotlab/src/robotlab/examples.py:17]: @datakurre maybe it's this?

File: robotlab/src/robotlab/tutorial.py [hunk 2]
```diff
@@ -14,7 +14,7 @@ def copy_robotkernel_tutorial(dest=None):
 
 if __name__ == "__main__":
     if len(sys.argv) > 1:
-        dest = Path(sys.argv(1))
+        dest = Path(sys.argv[1])
```
Comment (bollwyvl) [inline robotlab/src/robotlab/tutorial.py:17]: and this...

File: recipes/robotlab/meta.yaml [hunk 3]
```diff
@@ -18,7 +18,9 @@ build:
   entry_points:
     - robotlab = robotlab.labapp:main
     - robotlab-extension = robotlab.labextensionapp:main
+    - robotlab-shortcuts = robotlab.shortcuts:make_shortcuts
     - robotlab-examples = robotlab.examples:copy_robotkernel_examples
+    - robotlab-tutorial = robotlab.tutorial:copy_robotkernel_tutorial
```
Comment (bollwyvl) [inline recipes/robotlab/meta.yaml:23]: yeah, this is probably the problem! it was falling back to the `setuptools` ones, which generate scripts which don't work so well with conda.

**Comment by bollwyvl** [comment]
Great, that fixed tutorial...

Were your saying you were going to take a look at the other tests? I can as
well... When next at keyboard. Also is looks like some of the Screenshots
are getting messed up... I Probably need to do some reorganization of the
output directories.

**Comment by bollwyvl** [comment]
Doing some work locally on this: a big reason we might be getting mysterious errors during acceptance is all of the new `noarch:python` packages from `anaconda` (they used to not distribute any). From the install log, these are _certainly_ `noarch: python`:
```
installing: defusedxml-0.6.0-py_0 ...
installing: genson-1.1.0-py_0 ...
installing: parso-0.4.0-py_0 ...
installing: pytz-2019.1-py_0 ...
installing: strict-rfc3339-0.7-py_1 ...
installing: pygments-2.4.0-py_0 ...
installing: nbconvert-5.5.0-py_0 ...
```

I'll go ahead and build these, too, but this is getting a bit silly, and definitely starts to point at why my little constructor pre/post-processor will be a Good Thing To Open Source. I took a little detour to work some more of the model-based testing angle, and didn't do the testing/docs work we require to get stuff out the door, even though it's already quite useful. I'll push harder to get the azure/pypi/npm account stuff cleared up that's still blocking `irobotframework` (first email asking about that was sent from Helsinki :rofl:) and try to get that out!

**Comment by bollwyvl** [comment]
ok, built those, and got a clean build locally... :crossed_fingers: :neckbeard: :crossed_fingers:

**Comment by datakurre** [comment]
:-o on how hard this is...

I haven’t looked into tests yet :(

**Comment by bollwyvl** [comment]
> :-o on how hard this is...

Yeah, it's not all :rose:s, that's for sure.

**Comment by bollwyvl** [comment]
Found a [bad selector](https://github.com/robots-from-jupyter/robotframework-jupyterlibrary/issues/15) in `robotframework-jupyterlibrary`, seems good locally...

**Comment by bollwyvl** [comment]
Nope, that didn't do it... will have a look later...

**Comment by bollwyvl** [comment]
I'll take [one :green_heart:](
https://dev.azure.com/robots-from-jupyter/robots-from-jupyter/_build/results?buildId=137)!

**Comment by datakurre** [comment]
Replaced with https://github.com/robots-from-jupyter/robotlab/pull/41

**Pull request closed without merging**